src/com/facebook/buck/features/apple/project/SchemeGenerator.java

```diff
@@ -387,6 +399,14 @@ public static Element serializeEnvironmentVariables(
     return rootElement;
   }
 
+  public static Element serializeExpandVariablesBasedOn(
+    Document doc, XCScheme.BuildableReference reference) {
+    Element referenceElement = serializeBuildableReference(doc, reference);
+    Element rootElement = doc.createElement("MacroExpansion");
```

Comment:
"MacroExpansion" is used in apple's open source so it ok to use in buck generated scheme.
https://opensource.apple.com/source/Security/Security-57337.20.44/Security.xcodeproj/xcshareddata/xcschemes/Debug.xcscheme.auto.html